app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -153,7 +153,7 @@ The subject you apply to teach must usually be the same as, or closely related t
 
 ### Finding the right course for you and getting help
 
-[Find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) lists courses leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts).
+[Find postgraduate teacher training courses](https://find-teacher-training-courses.service.gov.uk/) lists courses leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts).
```

Comment:
I think the Find links on this page are a bit messy and not good practice - let's discuss!